Library/Formula/xorg-util-macros.rb

```diff
@@ -0,0 +1,19 @@
+class XorgUtilMacros < Formula
+  desc "Miscellaneous utility macros required by the X.org codebase."
+  homepage "http://www.x.org"
+  url "http://xorg.freedesktop.org/archive/individual/util/util-macros-#{version}.tar.gz"
```

Comment:
I'm fully aware of that but when you invert the lines the audit step generate a failure.

It is still working at runtime.